src/upload/UploadPage.test.jsx

```diff
@@ -62,6 +63,7 @@ describe('UploadPage...', () => {
     expect(tree).toMatchSnapshot();
     apiData.forEach((program, idx) => {
       expect(wrapper.find('h2').at(idx).text()).toEqual(program.programTitle);
+      expect(wrapper.find(Collapsible).at(idx).prop('defaultOpen')).toEqual(true);
```

Comment:
Are you thinking about a test case where the collapsible is not at default open state? Do we need to unit test here the fact that some elements should be default closed off?

Reply:
Maybe. The reason I included this was because we want the first collapsible of each program section to be open by default. For right now, each program will only have an enrollments collapsible. When we add in the reports collapsible, we may need to modify this statement to account for the fact that some programs' first collapsible will be an enrollments collapsible and some will be a reports collapsible. Does that answer your question?

Reply:
Ah. Make sense. That can come later. I agree